diff --git a/shared/config.gradle b/shared/config.gradle
index 02c2b687..6b4d092f 100644
--- a/shared/config.gradle
+++ b/shared/config.gradle
@@ -114,7 +114,7 @@ if (!project.hasProperty('skipAthena') && !project.hasProperty('onlyRaspbian'))
             roboRio(CrossBuildConfig) {
                 architecture = 'athena'
                 operatingSystem = 'linux'
-                toolChainPrefix = 'arm-frc2019-linux-gnueabi-'
+                toolChainPrefix = 'arm-frc-linux-gnueabi-'
                 compilerArgs = linuxCrossCompilerArgs
                 CCompilerArgs = linuxCrossCCompilerArgs
                 linkerArgs = linuxCrossLinkerArgs
