--- wpi-dependencies-2018.gradle.modified	2018-07-28 21:50:21.553404618 -0700
+++ wpi-dependencies-2018.gradle	2018-07-28 21:51:03.602369402 -0700
@@ -71,7 +71,7 @@
 
 ext.defineWpiUtilProperties = {
     ext.wpiUtil = wpiUtilUnzipLocation
+    ext.wpiUtilInclude = "$rootDir/../../../wpilib/src/wpilib/build/dependencies/wpiutil-cpp/headers"
-    ext.wpiUtilInclude = "$wpiUtilUnzipLocation/include"
     ext.wpiUtilLibArmLocation = "$wpiUtilUnzipLocation/linux/athena/shared"
     ext.wpiUtilSharedLib = "$wpiUtilLibArmLocation/libwpiutil.so"
     ext.wpiUtilSharedLibDebug = "$wpiUtilLibArmLocation/libwpiutil.so.debug"
@@ -302,7 +301,7 @@
 // This defines a project property that projects depending on network tables can use to setup that dependency.
 ext.defineNetworkTablesProperties = {
     ext.netTables = netTablesUnzipLocation
+    ext.netTablesInclude = "$rootDir/../../../wpilib/src/wpilib/build/dependencies/ntcore-cpp/headers"
-    ext.netTablesInclude = "$netTablesUnzipLocation/include"
     ext.netLibArmLocation = "$netTablesUnzipLocation/linux/athena/shared"
     ext.netSharedLib = "$netLibArmLocation/libntcore.so"
     ext.netSharedLibDebug = "$netLibArmLocation/libntcore.so.debug"
@@ -393,7 +392,7 @@
 
 ext.defineWpiLibProperties = {
     ext.wpilib = wpilibUnzipLocation
+    ext.wpilibInclude = "$rootDir/../../../wpilib/src/wpilib/wpilibc/src/main/native/include"
-    ext.wpilibInclude = "$wpilibUnzipLocation/include"
     ext.wpilibLocation = "$wpilibUnzipLocation/linux/athena/shared"
     ext.wpilibSharedLib = "$wpilibLocation/libwpilibc.so"
     
