diff --git a/shared/config.gradle b/shared/config.gradle
index d5ac9b17..055daba9 100644
--- a/shared/config.gradle
+++ b/shared/config.gradle
@@ -1,38 +1,39 @@
 import edu.wpi.first.nativeutils.*
 import org.gradle.internal.os.OperatingSystem
 
-def windowsCompilerArgs = ['/EHsc', '/D_CRT_SECURE_NO_WARNINGS', '/Zi', '/FS', '/Zc:inline', '/W3', '/wd4244', '/wd4267', '/wd4146', '/wd4996', '/std:c++17', '/Zc:throwingNew', '/permissive-', '/WX']
-def windowsCCompilerArgs = ['/Zi', '/FS', '/Zc:inline', '/W3', '/WX']
+def windowsCompilerArgs = ['/EHsc', '/D_CRT_SECURE_NO_WARNINGS', '/Zi', '/FS', '/Zc:inline', '/W3', '/wd4244', '/wd4267', '/wd4146', '/wd4996', '/std:c++17', '/Zc:throwingNew', '/permissive-', '/Imingw32\\include', "/I${projectDir}\\..\\..\\..\\..\\ni-libraries\\src\\ni-libraries\\src\\include"]
+
+def windowsCCompilerArgs = ['/Zi', '/FS', '/Zc:inline', '/W3']
 def windowsReleaseCompilerArgs = ['/O2', '/MD']
 def windowsDebugCompilerArgs = ['/Od', '/MDd']
 def windowsLinkerArgs = ['/DEBUG:FULL']
-def windowsReleaseLinkerArgs = ['/OPT:REF', '/OPT:ICF']
+def windowsReleaseLinkerArgs = ['/OPT:REF', '/OPT:ICF', "${projectDir}\\..\\..\\..\\..\\..\\Debug\\hel.lib", 'libwinpthread-1.lib']
 
-def linuxCrossCompilerArgs = ['-std=c++17', '-Wformat=2', '-Wall', '-Wextra', '-Werror', '-pedantic', '-Wno-psabi', '-g',
+def linuxCrossCompilerArgs = ['-std=c++14', '-Wformat=2', '-Wall', '-Wextra',  '-pedantic', '-Wno-psabi', '-g',
                          '-Wno-unused-parameter', '-Wno-error=deprecated-declarations', '-fPIC', '-rdynamic',
                          '-pthread']
-def linuxCrossCCompilerArgs = ['-Wformat=2', '-Wall', '-Wextra', '-Werror', '-pedantic', '-Wno-psabi', '-g',
+def linuxCrossCCompilerArgs = ['-Wformat=2', '-Wall', '-Wextra',  '-pedantic', '-Wno-psabi', '-g',
                           '-Wno-unused-parameter', '-fPIC', '-rdynamic', '-pthread']
 def linuxCrossLinkerArgs = ['-rdynamic', '-pthread', '-ldl']
 def linuxCrossReleaseCompilerArgs = ['-O2']
 def linuxCrossDebugCompilerArgs = ['-Og']
 
-def linuxCompilerArgs = ['-std=c++17', '-Wformat=2', '-Wall', '-Wextra', '-Werror', '-pedantic', '-Wno-psabi', '-g',
+def linuxCompilerArgs = ['-std=c++14', '-Wformat=2', '-Wall', '-Wextra',  '-pedantic', '-Wno-psabi', '-g',
                          '-Wno-unused-parameter', '-Wno-error=deprecated-declarations', '-fPIC', '-rdynamic',
                          '-pthread']
-def linuxCCompilerArgs = ['-Wformat=2', '-Wall', '-Wextra', '-Werror', '-pedantic', '-Wno-psabi', '-g',
+def linuxCCompilerArgs = ['-Wformat=2', '-Wall', '-Wextra',  '-pedantic', '-Wno-psabi', '-g',
                           '-Wno-unused-parameter', '-fPIC', '-rdynamic', '-pthread']
 def linuxLinkerArgs = ['-rdynamic', '-pthread', '-ldl']
 def linuxReleaseCompilerArgs = ['-O2']
 def linuxDebugCompilerArgs = ['-O0']
 def linux32BitArg = '-m32'
 
-def macCompilerArgs = ['-std=c++17', '-Wall', '-Wextra', '-Werror', '-pedantic-errors', '-fPIC', '-g',
+def macCompilerArgs = ['-std=c++14', '-Wall', '-Wextra',  '-pedantic-errors', '-fPIC', '-g',
                        '-Wno-unused-parameter', '-Wno-error=deprecated-declarations', '-Wno-missing-field-initializers',
                        '-Wno-unused-private-field', '-Wno-unused-const-variable', '-pthread']
-def macCCompilerArgs = ['-Wall', '-Wextra', '-Werror', '-pedantic-errors', '-fPIC', '-g',
+def macCCompilerArgs = ['-Wall', '-Wextra',  '-pedantic-errors', '-fPIC', '-g',
                         '-Wno-unused-parameter', '-Wno-missing-field-initializers', '-Wno-unused-private-field']
-def macObjCLinkerArgs = ['-std=c++17', '-stdlib=libc++','-fobjc-arc', '-g', '-fPIC', '-Wall', '-Wextra', '-Werror']
+def macObjCLinkerArgs = ['-std=c++14', '-stdlib=libc++','-fobjc-arc', '-g', '-fPIC', '-Wall', '-Wextra']
 def macReleaseCompilerArgs = ['-O2']
 def macDebugCompilerArgs = ['-O0']
 def macLinkerArgs = ['-framework', 'CoreFoundation', '-framework', 'AVFoundation', '-framework', 'Foundation', '-framework', 'CoreMedia', '-framework', 'CoreVideo']
