diff --git a/hal/src/main/native/athena/DigitalInternal.cpp b/hal/src/main/native/athena/DigitalInternal.cpp
index 89839602..bdc667c0 100644
--- a/hal/src/main/native/athena/DigitalInternal.cpp
+++ b/hal/src/main/native/athena/DigitalInternal.cpp
@@ -178,6 +178,6 @@ int32_t remapSPIChannel(int32_t channel) { return channel - 26; }
 }  // namespace hal
 
 // Unused function here to test template compile.
-__attribute__((unused)) static void CompileFunctorTest() {
+static void CompileFunctorTest() {
   hal::UnsafeManipulateDIO(0, nullptr, [](hal::DIOSetProxy& proxy) {});
 }
