diff --git a/cpp.gradle b/cpp.gradle
index 971db54..418e82a 100644
--- a/cpp.gradle
+++ b/cpp.gradle
@@ -14,7 +14,7 @@ def cppSetupModel = { project ->
 
                 binaries.all {
                     tasks.withType(CppCompile) {
-                        cppCompiler.args << "-DNAMESPACED_WPILIB" << "-Werror" << "-Wall"
+                        cppCompiler.args << "-DNAMESPACED_WPILIB" << "-Wall"
                         addUserLinks(linker, targetPlatform, false)
                         addHalLibraryLinks(it, linker, targetPlatform)
                         addWpiUtilLibraryLinks(it, linker, targetPlatform)
diff --git a/cpp/include/ctre/phoenix/MotorControl/CAN/WPI_TalonSRX.h b/cpp/include/ctre/phoenix/MotorControl/CAN/WPI_TalonSRX.h
index 6174ce8..2ad123e 100644
--- a/cpp/include/ctre/phoenix/MotorControl/CAN/WPI_TalonSRX.h
+++ b/cpp/include/ctre/phoenix/MotorControl/CAN/WPI_TalonSRX.h
@@ -111,7 +111,7 @@ public:
 
 	void SetSafetyEnabled(bool enabled);
 
-	void GetDescription(llvm::raw_ostream& desc) const;
+	void GetDescription(wpi::raw_ostream& desc) const;
 
 protected:
 	virtual void InitSendable(frc::SendableBuilder& builder);
diff --git a/cpp/include/ctre/phoenix/MotorControl/CAN/WPI_VictorSPX.h b/cpp/include/ctre/phoenix/MotorControl/CAN/WPI_VictorSPX.h
index f0c8cd0..d0b4355 100644
--- a/cpp/include/ctre/phoenix/MotorControl/CAN/WPI_VictorSPX.h
+++ b/cpp/include/ctre/phoenix/MotorControl/CAN/WPI_VictorSPX.h
@@ -111,7 +111,7 @@ public:
 
 	void SetSafetyEnabled(bool enabled);
 
-	void GetDescription(llvm::raw_ostream& desc) const;
+	void GetDescription(wpi::raw_ostream& desc) const;
 
 protected:
 	virtual void InitSendable(frc::SendableBuilder& builder);
diff --git a/cpp/src/MotorControl/CAN/WPI_TalonSRX.cpp b/cpp/src/MotorControl/CAN/WPI_TalonSRX.cpp
index 30a8bc9..52e4f3e 100644
--- a/cpp/src/MotorControl/CAN/WPI_TalonSRX.cpp
+++ b/cpp/src/MotorControl/CAN/WPI_TalonSRX.cpp
@@ -145,7 +145,7 @@ void WPI_TalonSRX::SetSafetyEnabled(bool enabled) {
 	_safetyHelper.SetSafetyEnabled(enabled);
 }
 
-void WPI_TalonSRX::GetDescription(llvm::raw_ostream& desc) const {
+void WPI_TalonSRX::GetDescription(wpi::raw_ostream& desc) const {
 	desc << _desc.c_str();
 }
 
diff --git a/cpp/src/MotorControl/CAN/WPI_VictorSPX.cpp b/cpp/src/MotorControl/CAN/WPI_VictorSPX.cpp
index cc8ddbd..bce1fd2 100644
--- a/cpp/src/MotorControl/CAN/WPI_VictorSPX.cpp
+++ b/cpp/src/MotorControl/CAN/WPI_VictorSPX.cpp
@@ -145,7 +145,7 @@ void WPI_VictorSPX::SetSafetyEnabled(bool enabled) {
 	_safetyHelper.SetSafetyEnabled(enabled);
 }
 
-void WPI_VictorSPX::GetDescription(llvm::raw_ostream& desc) const {
+void WPI_VictorSPX::GetDescription(wpi::raw_ostream& desc) const {
 	desc << _desc.c_str();
 }
 
diff --git a/gradlew b/gradlew
old mode 100644
new mode 100755
diff --git a/toolchains/arm.gradle b/toolchains/arm.gradle
index 4fe9f86..200ac12 100644
--- a/toolchains/arm.gradle
+++ b/toolchains/arm.gradle
@@ -23,7 +23,7 @@ model {
                 // Gradle auto-adds the -m32 argument to the linker and compiler. Our compiler only supports
                 // arm, and doesn't understand this flag, so it is removed from both
                 cppCompiler.withArguments { args ->
-					args << '-std=c++1y' << '-Wformat=2' << '-Wall' << '-Wextra' << '-Werror' << '-pedantic'
+					args << '-std=c++1y' << '-Wformat=2' << '-Wall' << '-Wextra' << '-pedantic'
                     args << '-fPIC' << '-rdynamic'
                     args << '-pthread'
                     args.remove('-m32')
diff --git a/wpi-dependencies-2018.gradle b/wpi-dependencies-2018.gradle
index b38190e..947bdee 100644
--- a/wpi-dependencies-2018.gradle
+++ b/wpi-dependencies-2018.gradle
@@ -71,7 +71,7 @@ task unzipWpiUtilHeaders(type: Copy) {
 
 ext.defineWpiUtilProperties = {
     ext.wpiUtil = wpiUtilUnzipLocation
-    ext.wpiUtilInclude = "$wpiUtilUnzipLocation/include"
+    ext.wpiUtilInclude = "$rootDir/../../../wpilib/src/wpilib/wpiutil/src/main/native/include"
     ext.wpiUtilLibArmLocation = "$wpiUtilUnzipLocation/linux/athena/shared"
     ext.wpiUtilSharedLib = "$wpiUtilLibArmLocation/libwpiutil.so"
     ext.wpiUtilSharedLibDebug = "$wpiUtilLibArmLocation/libwpiutil.so.debug"
@@ -301,7 +301,7 @@ task unzipNetworkTablesHeaders(type: Copy) {
 // This defines a project property that projects depending on network tables can use to setup that dependency.
 ext.defineNetworkTablesProperties = {
     ext.netTables = netTablesUnzipLocation
-    ext.netTablesInclude = "$netTablesUnzipLocation/include"
+    ext.netTablesInclude = "$rootDir/../../../wpilib/src/wpilib/ntcore/build/main/native/include"
     ext.netLibArmLocation = "$netTablesUnzipLocation/linux/athena/shared"
     ext.netSharedLib = "$netLibArmLocation/libntcore.so"
     ext.netSharedLibDebug = "$netLibArmLocation/libntcore.so.debug"
@@ -392,7 +392,7 @@ task unzipWpilibHeaders(type: Copy) {
 
 ext.defineWpiLibProperties = {
     ext.wpilib = wpilibUnzipLocation
-    ext.wpilibInclude = "$wpilibUnzipLocation/include"
+    ext.wpilibInclude = "$rootDir/../../../wpilib/src/wpilib/wpilibc/src/main/native/include"
     ext.wpilibLocation = "$wpilibUnzipLocation/linux/athena/shared"
     ext.wpilibSharedLib = "$wpilibLocation/libwpilibc.so"
     
